<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>377</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferDefault</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>Gadugi</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">align-items: center;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <property name="styleSheet">
        <string notr="true">align-items: center;</string>
       </property>
       <attribute name="title">
        <string>Add New Tasks</string>
       </attribute>
       <widget class="QLabel" name="label_start_time">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Gadugi</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Starting Time:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_end_time">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>21</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Gadugi</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Ending Time:</string>
        </property>
       </widget>
       <widget class="QTimeEdit" name="input_start_time">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>38</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Gadugi</family>
         </font>
        </property>
       </widget>
       <widget class="QTimeEdit" name="input_end_time">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>38</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Gadugi</family>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>21</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Gadugi</family>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Date:</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="input_date">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>38</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Gadugi</family>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="label_task_desc">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>112</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Gadugi</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Task Description (optional):</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_types_of_tasks">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>70</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Gadugi</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Select a Task:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comBox_tasks_types">
        <property name="geometry">
         <rect>
          <x>41</x>
          <y>90</y>
          <width>311</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Gadugi</family>
         </font>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="input_task_desc">
        <property name="geometry">
         <rect>
          <x>41</x>
          <y>131</y>
          <width>391</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Gadugi</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="button_save">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>190</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Gadugi</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Save Task</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="button_add_type">
        <property name="geometry">
         <rect>
          <x>355</x>
          <y>88</y>
          <width>31</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button_delete_type">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>88</y>
          <width>31</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Display Registered Tasks</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <widget class="QPushButton" name="button_del_task">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Delete selected task</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTableWidget" name="done_tasks_table">
          <property name="cursor" stdset="0">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideRight</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="columnCount">
           <number>4</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>112</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <column/>
          <column/>
          <column/>
          <column/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
